core/language/en-GB/Fields.multids

```diff
@@ -3,6 +3,7 @@ title: $:/language/Docs/Fields/
 _canonical_uri: The full URI of an external image tiddler
 bag: The name of the bag from which a tiddler came
 caption: The text to be displayed on a tab or button
+class: The CSS class applied to a tiddler when rendering it - see [[Custom styles by user-class]].  Also used for [[Modals]]
```

Comment:
This file is part of the core plugin, and so will have to be updated via the master branch, and wait for the next version before release.

Reply:
Ok, thanks for the feedback @Jermolene!  What's the right way to proceed here?  Do I need to commit on what's currently on master and submit an PR against the master branch, then?  Or will my change just make it over into master the next time you merge `tiddlywiki-com` into master?

Reply:
> Do I need to commit on what's currently on master and submit an PR against the master branch, then?

Correct.

Reply:
Ok, done! I opened #6990 against `master`

Reply:
BTW, if I have more questions about the workflow for contributing to the docs (eg. which tiddlers dictate the source/target branch), would you prefer I ask in https://github.com/Jermolene/TiddlyWiki5/discussions, talk.tiddlywiki.org, or somewhere else?

Reply:
Github discussions would be good I think. Generally anything that is a part of the core itself (so the boot and core directory in the root of the repo and their contents) need to be changed as part of a TW release and therefore PRs need to be against master. The same applies for purely documentation updates that have to do with new features that are not yet part of a released TW version.

Documentation PRs that target existing features and tiddlers in the editions directory can go live immediately and should target the tiddlywiki-com branch.